### PR TITLE
Update ray-values.yaml

### PR DESCRIPTION
**Pull request by idanb1**
Added fix for head pod getting GPUs where it shouldn't:
- name: NVIDIA_VISIBLE_DEVICES value: "void" Fix is based on this ray-project public repo issue: TLDR - looks like GPU operator bypassing gpus to head pod; https://github.com/ray-project/ray/issues/29753

Added nodeSelectors for redis and head pod, so by default they will 'eat' resources only from CPU nodes, based on labels: beta.kubernetes.io/instance-type: standard-v3

### CODE REVIEW

**Comment by idanb1** [comment]
@malibora @d3vil-st  Guys, Sutter Hill, and probably Genesis will use our kuberay solution from Oldbius soon - So it's really important to merge this PR asap, it's very hard to follow all of those multiple commits seperately.

**Review by malibora — APPROVED** [review]
